### DISCUSSION TITLE
How many conferences are there?

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#241] [General]
http://ptp.bitplan.com/settings shows records from a few sources that might overlap.

There are at least 86000 WikiCFP call for papers in the meantime. The TIBKAT / GND data contains at least 630.000 proceedings - depending on the proceedings / event ratio i'd estimate that there are some 500.000 scientific events with digital traces. Getting a more precise answer to this question is part of my research.

**Comment by angelosalatino** [comment]
Hi Wolfgang, knowing/estimating the ratio between proceedings and digital events is great, however as a number, it is limited to what it represents. It would be interesting to see the distribution. I am assuming (until debunked) that there is a long tail of event which took place only once. Having a distribution allows us also to see how many conferences are successful (i.e. they run/are running over a sustained period of time). What do you think?

**Comment by WolfgangFahl** [comment]
@angelosalatino  my ratio interpretation was a bit different then yours - i'll get back to this futher down. As for your assumed long tail distribution this is the result for wikidata scientific papers with "Proceedings of" in the name and the CEUR-WS volumes - in total 8471 events are covered by this statistic: 
![grafik](https://user-images.githubusercontent.com/1336221/124939960-d8b4ff00-e009-11eb-9fae-9a703cf4c827.png)


For these two proceedings title parser sources I am already extracting the "ordinal" information. Most other sources do not have that attribute as an explicit field. Still your long tail assumption is correct and most events are 1st or 2nd events while only a few event series make it to ten ore more installments. 

My own ratio intepretation was about how many volumes a proceedings publication may have. In the GND / TIBKAT dataset each volume is considered to be its own "Tagungsband".  My [favorite query on TIBKAT](https://www.tib.eu/de/suchen?tx_tibsearch_search%5Baction%5D=search&tx_tibsearch_search%5Bcnt%5D=20&tx_tibsearch_search%5Bcontroller%5D=Search&tx_tibsearch_search%5BDdocumentType%5D%5B0%5D=c&tx_tibsearch_search%5Bpg%5D=1&tx_tibsearch_search%5Bquery%5D=prefix%3A%28tibkat%29&cHash=7bc92b00eb26fe3e06d3ca33420d90a9) shows a diagram 
![grafik](https://user-images.githubusercontent.com/1336221/124939497-6fcd8700-e009-11eb-96a3-06dc7f065092.png)
that counts entries per volume. I assume most proceedings of events are single volume but i don't have any empirical evidence for the ratio yet since i am still struggling to get systematic access to the TIBKAT dataset.

**Reply by WolfgangFahl** [reply]
So i assume your interpretation of "How many conferences are there" is "How many scientific event series are there?".

In may I did a delta comparison of how many new events and series showed up in the ptp sources in the last few month:
![grafik](https://user-images.githubusercontent.com/1336221/124958092-0f931100-e01a-11eb-81c1-58fff18d2b0c.png)

The highest number of series is 9564 in WikiCfP with an increase of 13 series per week. All sources overlap and i didn't consolidate yet by doing proper entity matching yet.

Note that the big increase in wikidata series entries is due to @Simon-Cobb 's work on #127

**Comment by birukou** [comment]
@WolfgangFahl , I was wondering if you could also easily generate ordinal distribution from Confref data (where conference number in the series is stored in a dedicated field, so it could be considered gold standard) or from the DBLP (who would have bigger coverage than confref, i.e. would index first events published in CEUR-WS which later came to Springer). 
Overall, nice to see you working on this topic and looking forward to our cooperation. I have downloaded the draft and will aim at sending my comments this week.
One of "fun" questions is - among all entries you have, is there the first chemistry conference? https://en.wikipedia.org/wiki/Karlsruhe_Congress. Would we see areas where proceedings are not popular underrepresented? Any duplicates in different languages, like 

- XXII International Conference on Distributed Computer and Communication Networks: Control, Computation, Communications. 23-27 September 2019, Moscow, Russia
- XXII Международная конференция "Распределенные компьютерные и телекоммуникационные сети: управление, вычисление, связь", 23-27 сентября 2019 г.

**Comment by birukou** [comment]
@WolfgangFahl , I read the draft with interest. For the Gold standard, I'd recommend mentioning DBLP, which was de-facto standard since 1990s and also the LOD pilot from Springer - as it was nicely received by the community. I can dig up more facts (and put them on Arxiv since we never formally published the extended version). The preliminary version is at http://ceur-ws.org/Vol-1155/paper-01.pdf and an evolved (but short) at http://ceur-ws.org/Vol-1963/paper566.pdf

**Comment by birukou** [comment]
Also, we worked on the dataset of peer review descriptions to make the peer review process more transparent - see https://www.slideshare.net/birukou/creating-a-dataset-of-peer-review-in-computer-science-conferences-published-by-springer, also unpublished
